web-registry/src/App.tsx

```diff
@@ -103,14 +103,30 @@ Sentry.init({
 
 export const history = createBrowserHistory();
 
-const GoogleAnalytics: React.FC = (): JSX.Element => {
-  useGoogleAnalyticsInit();
+const PageViewTracking: React.FC = (): JSX.Element => {
```

Comment:
This is a component that sends page views using the `analytics` hook. This will send page views to all registered analytics `plugins` (so in our case amplitude and google analytics). This setup is what allows deprecation of the previous GA page view setup.